config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -126,25 +126,32 @@ presubmits:
       preset-azure-cred-only: "true"
       preset-azure-anonymous-pull: "true"
       preset-capz-windows-2022: "true"
-      preset-capz-containerd-1-7-latest: "true"
       preset-capz-windows-common-pull: "true"
+      preset-windows-private-registry-cred: "true"
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
         base_ref: release-1.9
         path_alias: sigs.k8s.io/cluster-api-provider-azure
-        workdir: true
+        workdir: false
       - org: kubernetes-sigs
         repo: cloud-provider-azure
         base_ref: master
         path_alias: sigs.k8s.io/cloud-provider-azure
         workdir: false
+      - org: kubernetes-sigs
+        repo: windows-testing
+        base_ref: master
+        path_alias: k8s.io/windows-testing
```

Comment:
should this be `sigs.k8s.io/windows-testing`?

Reply:
maybe, I don't think it matters since it is the `default` and not referenced anywhere.  I copied from https://github.com/kubernetes/test-infra/blob/40d95becb72f10cf73526b09a307099b66e79f3b/config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml#L90 which has the same value...

Reply:
we can update them all at once if needed then